articles/how-to-convert-text-to-emoji-using-python/index.md

```diff
@@ -0,0 +1,155 @@
+See example below
+
+```Python
+first_name='john'
+last_name='smith'
+message=f"{first_name} + {last_name} is a coder"
+print(message)
+country='nigeria is the giant of Africa'
+locate='and it is a West African country'
+msg=f'{country} +  {locate} and that is thriving to be the best in the world'
+print(msg)
+```
+
+### IF and ELSE statement
+The IF and ELSE statement allows us to build programs that can decide based on certain conditions (either these conditions are TRUE or FALSE). They test a condition and then return a value depending on the outcome. The value returned depends on whether the condition passes or fails.
+
+```Python
+is_hot=False
+is_cold=False
+
+if is_hot:
+   print("its a hot day")
+   print("please drink plenty of water")
+elif is_cold:
+   print("its a cold day")
+   print("please waer something warm")
+else:
+   print("its a lovely day")
+print("have a nice day ahead")
+```
+
+### The input and print function
+The print and input are general-purpose functions that are built into Python. You can use the print function to print a specified message on the screen. This message can be a string or any other object, while you can use the input function to receive input/messages from the user. This function can instruct the program to pause and wait for the user to enter the required information. See the example below.
+
+```Python
+name=input("what is your name? ")
+age=int(input("how old are you?  " + name ))
+favourite_colour=input("what is your favorite colour? ")
+print(name + " likes " + favourite_colour)
+```
+
+Check your terminal to see the result.
+
+### Dictionary
+Dictionaries stores data values in key pairs. A dictionary is a list of objects that can be rearranged without repeating its data values. You write the keys and values in dictionaries in curly brackets, it is an ordered list of data values. The key name refers to dictionary objects, which are presented in keys; value pairs.
+
+```Python
+details = { "brand": "Ford",
+           "model": "Mustang",
+           "year" : 2001}
+print(details.get("brand"))
+print(details.get("model"))
+print(details.get("year"))
+```
+
+In the above illustration, you can use the `.get()` method to get values stored in a dictionary.
+The  `.update()` methods will update the items from an argument. If the item does not exist, then it will add the item to the dictionary. Other methods you can use are:
+`.pop()` this method removes the item with the specific key name
+`.del()` removes the item with the specified key name
+`.clear()`method empties the dictionary
+Try more tricks using the  `dictionary.get()` method.
+Kindly try this yourself.
```

Comment:
Check the formatting. If you intend to type this on a new line, then you need to use two enters. View the file and check if the formatting suits your liking.